hdrass/CCW+Rotator_integration_testing/Rotator_interlock_test.ipynb

```diff
@@ -0,0 +1,1007 @@
+    "    -------\n",
+    "    `~astropy.coordinates.Angle`\n",
+    "        Parallactic angle.\n",
+    "    Notes\n",
+    "    -----\n",
+    "    The parallactic angle is the angle between the great circle that\n",
+    "    intersects a celestial object and the zenith, and the object's hour\n",
+    "    circle [1]_.\n",
+    "    .. [1] https://en.wikipedia.org/wiki/Parallactic_angle\n",
+    "    \"\"\"\n",
+    "    # Eqn (14.1) of Meeus' Astronomical Algorithms\n",
+    "    H = (lst - target.ra).radian\n",
+    "    q = np.arctan2(np.sin(H),\n",
+    "                   (np.tan(location.lat.radian) *\n",
+    "                    np.cos(target.dec.radian) -\n",
+    "                    np.sin(target.dec.radian)*np.cos(H)))*u.rad\n",
+    "    return Angle(q)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "test_message = \"Interlock Rotator_CCW Integration Test - LVV-T1569\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "test_message = \"Interlock Rotator_CCW Integration Test - limit switch characterisation\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "d = salobj.Domain()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "rotation DDS read queue is full (100 elements); data may be lost\n",
+      "motors DDS read queue is full (100 elements); data may be lost\n",
+      "electrical DDS read queue is full (100 elements); data may be lost\n",
+      "ccwFollowingError DDS read queue is full (100 elements); data may be lost\n",
+      "timeAndDate DDS read queue is filling: 15 of 100 elements\n"
+     ]
+    }
+   ],
+   "source": [
+    "script = salobj.Controller(\"Script\", index=1)\n",
+    "rot = salobj.Remote(d, \"MTRotator\")\n",
+    "mtptg = salobj.Remote(d, \"MTPtg\")\n",
+    "mtm = salobj.Remote(d, \"MTMount\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "await asyncio.gather(rot.start_task,\n",
+    "                     mtptg.start_task,\n",
+    "                     script.start_task,\n",
+    "                     mtm.start_task)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": []
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "await asyncio.sleep(1.)\n",
+    "await salobj.set_summary_state(mtptg, salobj.State.ENABLED)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "await rot.cmd_clearError.start(timeout=10)"
```

Comment:
Warning; the current MTRotator release does not support the clearError command (it happens automatically as part of enabling the CSC). I don't know what version this code is written for so I don't know if you need this or not.

Reply:
Yes, this was for an older version of the MTRotator CSC. I leave it here since it belongs to the execution of a verification test case where the clearError command was still available. I added a warning to update this when we come back to test this with the new rotator tracjectory.